rockcraft/providers/_lxd.py

```diff
@@ -23,8 +23,6 @@
 
 from craft_providers import Executor, ProviderError, base, bases, lxd
 
-from rockcraft.utils import confirm_with_user
```

Comment:
:tada: :tada: - The last `rockcraft` imports have been pulled out of `_lxd.py`, `multipass.py`, and `_provider.py`!